src/types/geometry.ts

```diff
@@ -0,0 +1,54 @@
+export type Coordinates = [number, number];
+
+interface Geometry {
```

Comment:
Can you put a link to where these definitions come from?

Reply:
Thank you for reviewing my pull request. I have added types to the GEOJSON objects used in the geo-spatial queries. 
I sourced these definitions from the official MongoDB documentation, which can be found here: https://www.mongodb.com/docs/manual/reference/geojson/#std-label-geospatial-indexes-store-geojson. I hope this clarifies the changes I have made